pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -27,10 +28,15 @@ const (
 	libvirtDriverCommand     = "crc-driver-libvirt"
 	libvirtDriverDownloadURL = "https://github.com/code-ready/machine-driver-libvirt/releases/download/0.9.1/crc-driver-libvirt"
 	defaultPoolSize          = "20 GB"
+	crcDnsmasqConfigFile     = "crc.conf"
 )
 
 var (
 	libvirtDriverBinaryPath = filepath.Join(driverBinaryDir, libvirtDriverCommand)
+	crcDnsmasqConfigPath    = filepath.Join(string(filepath.Separator), "etc", "NetworkManager", "dnsmasq.d", crcDnsmasqConfigFile)
+	crcDnsmasqConfig        = `server=/tt.testing/192.168.126.1
+address=/apps.tt.testing/192.168.126.11
```

Comment:
Now that i see this again, will this always be static for linux/libvirt?

Reply:
For now... usuable as-is.